docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.4'
```

Comment:
Top-level version attribute is deprecated
https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file